client/lib/directly/test/index.js

```diff
@@ -36,65 +42,109 @@ describe( 'index', () => {
 		delete require.cache[ require.resolve( '..' ) ];
 	} );
 
-	describe( '#initialize()', () => {
-		it( 'creates a window.DirectlyRTM function', () => {
-			directly.initialize();
-			expect( typeof window.DirectlyRTM ).to.equal( 'function' );
+	describe( 'when the API says Directly is available', () => {
+		useNock( ( nock ) => {
+			nock( 'https://public-api.wordpress.com:443' )
+				.persist()
+				.get( '/rest/v1.1/help/directly/mine' )
+				.reply( 200, {
+					isAvailable: true
+				} );
 		} );
 
-		it( 'attempts to load the remote script', () => {
-			directly.initialize();
-			expect( loadScript.loadScript ).to.have.been.calledOnce;
-		} );
+		describe( '#initialize()', () => {
+			it( 'creates a window.DirectlyRTM function', ( done ) => {
+				directly.initialize()
+					.then( () => expect( typeof window.DirectlyRTM ).to.equal( 'function' ) )
+					.then( () => done() );
+			} );
 
-		it( 'does nothing after the first call', () => {
-			directly.initialize();
-			directly.initialize();
-			directly.initialize();
+			it( 'attempts to load the remote script', ( done ) => {
+				directly.initialize()
+					.then( () => expect( loadScript.loadScript ).to.have.been.calledOnce )
+					.then( () => done() );
+			} );
 
-			expect( window.DirectlyRTM.cq ).to.have.lengthOf( 1 );
-			expect( window.DirectlyRTM.cq[ 0 ][ 0 ] ).to.equal( 'config' );
-			expect( loadScript.loadScript ).to.have.been.calledOnce;
-		} );
+			it( 'does nothing after the first call', ( done ) => {
+				Promise.all( [
+					directly.initialize(),
+					directly.initialize(),
+					directly.initialize()
+						.then( () => {
+							expect( window.DirectlyRTM.cq ).to.have.lengthOf( 1 );
+							expect( window.DirectlyRTM.cq[ 0 ][ 0 ] ).to.equal( 'config' );
+							expect( loadScript.loadScript ).to.have.been.calledOnce;
+						} )
+				] ).then( () => done() );
```

Comment:
I think your `expect` logic should go here instead of being called on the `initialize` promise. so 
```
Promise.all( [
	directly.initialize(),
	directly.initialize(),
	directly.initialize()
] )
.then( () => {
	expect( window.DirectlyRTM.cq ).to.have.lengthOf( 1 );
	expect( window.DirectlyRTM.cq[ 0 ][ 0 ] ).to.equal( 'config' );
	expect( loadScript.loadScript ).to.have.been.calledOnce;
} )
.then( () => done() );
```